docs/source/documentation/team/our-ceremonies.html.md.erb

```diff
@@ -0,0 +1,48 @@
+---
+owner_slack: "#data-platform-notifications"
+title: Our Ceremonies
+last_reviewed_on: 2023-04-12
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+Our stand-ups run daily at 09:40 and last for fifteen minutes.  We will follow one of two formats:
+
+- **Walk The Board**
+
+    Starting with the "Done" column we discuss any active issues in GitHub via [ZenHub](https://app.zenhub.com/workspaces/data-platform-core-infrastructure-63e3ab98f777b90011c85bbb/board)
+
+- **Yesterday, Today, Blocked**
+
+    A round robin of each team member discussing their issues and any blockers.
+
+## Sprint Planning 
+
+Each Sprint is two weeks long and starts on a Thursday.
+We use Miro or Jamboard to run a one-hour session. Template agenda below:
```

Comment:
We just use the zenhub board for planning

Reply:
For sprint planning: 

1. Discuss the previous sprint and if we met our objective/estimated tickets, did we achieve anything outside of our goal? 
2. Review any tickets caring over to the next sprint and why
3. Agree team capacity for the sprint
4. Pull any new issues into the sprint backlog and ensure all tickets have sufficient descriptions, epics assigned and estimated size
5. AOB